source/includes/stackpath/_firewallrules.md

```diff
@@ -0,0 +1,94 @@
+## Firewall Rules
+
+Deploy and manage Firewall Rules used to control and limit access to your sites. 
+
+<!-------------------- LIST FIREWALL RULES -------------------->
+
+### List firewall rules
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/api/v1/services/stackpath/test-area/firewallrules?siteId=0a57855b-26d8-4e8f-8b77-429997c7c5fb"
+```
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "data": [
+    {
+        "id": "1533148",
+        "name": "Rule 1 ",
+        "ipStart": "10.10.10.10",
+        "ipEnd": "20.20.20.20",
+        "action": "ALLOW",
+        "enabled": "true"
+    },
+    {
+        "id": "1533345",
+        "name": "Rule 2 ",
+        "ipStart": "192.0.0.1",
+        "ipEnd": "192.1.1.2",
+        "action": "BLOCK",
+        "enabled": "true"
+    }
+  ],
+  "metadata": {
+    "recordCount": 2
+  }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/firewallrules?siteId=<a href="#stackpath-sites">:siteId</a></code>
+
+Query Params | &nbsp;
```

Comment:
@aaronalmeida 
This requires adding one more optional param - `action` which can filter the rules based on `ALLOW/BLOCK`

I'll do this change.

Reply:
@monil-panchal Sounds good!

Reply:
Added here - https://github.com/cloudops/cloudmc-api-docs/pull/339/commits/aea0eea9fca6ee06ae04b292b9490963f626be70

@aaronalmeida @razine-bensari Is it looking good or needs some changes?

Reply:
@monil-panchal Thanks, looks good to me!